### DISCUSSION TITLE
Permissions: _count operator in permissions?

### DISCUSSION THREAD

**Discussion by petegriffith** [hasura/graphql-engine#9233] [Question]
When setting custom permissions, is there a way to use something like _count to check the number of entries on a table?

For instance, a given user role has permission to delete an entry from the `special_user` table if they are in the special_user table AND there is at least one other `special_user` in the table. The first half is simple:
`{"special_users":{"account":{"auth_uid":{"_eq":"X-Hasura-User-Id"}}}}`

But what would the second half of the _and look like? Something like this?
`{"special_users":{_COUNT:{"_gt": 0}}`

**Comment by adas98012** [comment]
@petegriffith 
One alternative would be to use a [computed field](https://hasura.io/docs/latest/schema/postgres/computed-fields/) that holds the count (returned by a custom sql function), which you can then use in the permissions logic.
